src/Display/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import 'jest-styled-components';
```

Comment:
I think we can include `jest-styled-components` in all tests, so css can be included in snapshot.

Reply:
I thought that the classnames generated by styled-components are hashes of the CSS? In that case the classname would change when the CSS changes, and thus the snapshot comparison would fail.

Reply:
yes, my idea is that maybe we should check css in the first generated snapshot to see whether the css is correct, otherwise we can only detect css changes.

Reply:
Yeah but we can't check for correctness of the CSS. We can only make sure that it does not change unexpectedly.